aws_sqs.rb

```diff
@@ -0,0 +1,10 @@
+class AwsSQS
```

Comment:
Would be more explicit if we name the class `AwsSQSClient`  or something like that.

Reply:
I renamed it.. will push after all changes are done.